encoding/preparation.md

```diff
@@ -204,23 +204,14 @@ and which ones you want to use.
 
 The codec used most commonly is x264,
 an implementation of the h.264 standard.
-There are multiple versions available
-(kmod and tmod being the most popular ones),
-but they only differ slightly
-(mainly cosmetic patches and some obscure parameters
-that you will probably never use),
-and no version is objectively better than any other.
-x264 can encode in 8 or 10 bit color depth.
-We will explain the meaning of these at a later time.
-For now, they are just relevant for choosing the right binary.
-All modifications come in two versions each,
-one for 8 bit and one for 10 bit[^2].
-Most fansub group nowadays use 10 bit,
-so you should download that
-unless your group leader or encode mentor told you otherwise.
-
-- [Download kmod](http://komisar.gin.by/)
-- [Download tmod](https://github.com/jpsdr/x264/releases)
```

Comment:
Maybe leave in a short paragraph about the x264 mods and why they aren't needed anymore (ideally with a date for reference).

Reply:
A few days before this, so at some point in August 2018
https://github.com/jpsdr/x264/releases/tag/r2932

Reply:
Actually seems to be from much earlier, 24 Dec 2017. [Commit](https://git.videolan.org/?p=x264.git;a=commit;h=71ed44c7312438fac7c5c5301e45522e57127db4)

Reply:
I can't find an exact date for the commit, so I'm going to include August 2018 for now.